pkg/report/output/security/security.go

```diff
@@ -444,16 +461,27 @@ func CalculateSeverity(groups []string, severity string, hasLocalDataTypes bool)
 		triggerWeighting = 2
 	}
 
-	switch finalWeighting := ruleSeverityWeighting + (sensitiveDataCategoryWeighting * triggerWeighting); {
+	var displaySeverity string
+	finalWeighting := ruleSeverityWeighting + (sensitiveDataCategoryWeighting * triggerWeighting)
+	switch {
 	case finalWeighting >= 8:
-		return globaltypes.LevelCritical
+		displaySeverity = globaltypes.LevelCritical
 	case finalWeighting >= 5:
-		return globaltypes.LevelHigh
+		displaySeverity = globaltypes.LevelHigh
 	case finalWeighting >= 3:
-		return globaltypes.LevelMedium
+		displaySeverity = globaltypes.LevelMedium
+	default:
+		displaySeverity = globaltypes.LevelLow
 	}
 
-	return globaltypes.LevelLow
+	return SeverityWeighting{
+		RuleSeverity:                   severity,
+		SensitiveDataCategoryWeighting: sensitiveDataCategoryWeighting,
+		RuleSeverityWeighting:          ruleSeverityWeighting,
+		TriggerWeighting:               &hasLocalDataTypes,
```

Comment:
Suggestion to keep the terminology the same here: 

```suggestion
		TriggerWeighting:               triggerWeighting,
```

or 

```suggestion
		HasLocalDataTypes:               &hasLocalDataTypes,
```

Reply:
Make the changes. Good call